### PR TITLE
YAML based configuration

### PR DESCRIPTION
**Pull request by swiebertje**
PR for #12

### CODE REVIEW

**Comment by PMudra** [comment]
Looks great! I added two comments that we should discuss before I accept the PR.

**Comment by swiebertje** [comment]
Okay but i don't see your comments 😛

**Comment by PMudra** [comment]
My bad. They were still pending 😄 Now you should be able to see them.

File: ApiCheck.Test/Loader/ConfigurationLoaderTest.cs [hunk 1]
```diff
@@ -0,0 +1,57 @@
+﻿using System.IO;
+using System.Text;
+using ApiCheck.Configuration;
+using ApiCheck.Loader;
+using ApiCheck.Result.Difference;
+using NUnit.Framework;
+
+namespace ApiCheck.Test.Loader
+{
+  class ConfigurationLoaderTest
```
Comment (swiebertje) [inline ApiCheck.Test/Loader/ConfigurationLoaderTest.cs]: Sure I will add them.

File: ApiCheck.Test/Loader/ConfigurationLoaderTest.cs [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+            ignore:   
+              - MyCompany.MyNamespace.MyClass.SomeMember
+              - MyCompany.MyNamespace.MyClass2 # exclude MyClass2 because it changes a lot 
+
+            # override the default comparer severities
+            # the rule names match the descriptions from the table above in camelCase
+            # the severity levels can be set to error/warning/hint
+            severities:
+              parameterNameChanged: warning # globally set the check for changed parameter names to warning
+              assemblyNameChanged: hint";
```
Comment (swiebertje) [inline ApiCheck.Test/Loader/ConfigurationLoaderTest.cs]: I will move it to separate file.

**Comment by swiebertje** [comment]
This commit provides more tests for the configuration loader.

**Comment by PMudra** [comment]
Thanks for the contribution! 🥇

**Pull request merged**